config/projects/datadog-agent.rb

```diff
@@ -18,23 +18,40 @@
  forwards metrics from your applications as well as system services.
  .
  See http://www.datadoghq.com/ for more information
-"
-platform_in_iteration false
+'
+
+# .deb specific flags
+package :deb do
+  vendor 'Datadog <info@datadoghq.com>'
```

Comment:
I have to duplicate it since vendor is specific to the `package` DSL and not the `project` one. In addition to that, this DSL is defined for DEB and RPM packagers but not for the other ones (OSX, Windows...)